keps/sig-node/2400-node-swap/README.md

```diff
@@ -117,6 +118,21 @@ support to nodes in a controlled, predictable manner so that Kubernetes users
 can perform testing and provide data to continue building cluster capabilities
 on top of swap.
 
+Since swap enablement is crucial to many different parties, this KEP aims to
+introduce basic swap enablement and leave further extensions to follow-up KEPs,
+that are already being worked on and will be introduced in the very near future.
```

Comment:
This would get a bit stale over time. Let's drop this.

Reply:
this KEP aims to
introduce basic swap enablement and leave further extensions to follow-up KEPs.

Reply:
Makes sense, thank you!
Done.

Reply:
@kannon92 are you aware of further extensions and existing keps i can follow?

Reply:
@Siegfriedk Due to me being out of office for a while, opening the follow-up KEPs was delayed.
However, I will get to it soon.
Would you like me to cc you once I do?

Reply:
Yes pls :)

Reply:
Will do! :)